Assets/Tayx/Graphy - Ultimate Stats Monitor/Scripts/Fps/G_FpsMonitor.cs

```diff
@@ -37,11 +37,10 @@ public class G_FpsMonitor : MonoBehaviour
         private                     float           m_maxFps                    = 0f;
 
         private                     float[]         m_averageFpsSamples;
-        private                     int             m_avgFpsSamplesOffset       = 0;
-        private                     int             m_indexMask                 = 0;
```

Comment:
Can you double check removing this doesn't break the avg in the first frames before they reach m_averageFpsSamples.Lenght ?

Reply:
I did some calculation and it pretty much match the result with first 36 frame, just ignore the 0.01 imprecision cause I only input 2 number after coma ![image](https://i.imgur.com/wLKi71o.png)